src/java.base/share/classes/java/lang/foreign/Arena.java

```diff
@@ -77,12 +76,12 @@
  *
  * {@snippet lang = java:
  * MemorySegment segment = null;
- * try (Arena arena = Arena.ofConfined()) {
+ * try (Arena arena = Arena.ofConfined()) { // @highlight regex='ofConfined()'
  *     segment = arena.allocate(100);
- *     ...
+ *     // ...
```

Comment:
Do we need to add comments here? I believe that, even with hybrid snippets there are ways to handle this (I remember it being discussed)

Reply:
Same for all other occurrences of `// ...`. Let's address them when we have to (if we have to).